### PR TITLE
feat: Add `opts.SchedulerOpts()` helper to convert `plugin.SyncOptions` for scheduler

### PR DESCRIPTION
**Pull request by disq**

### CODE REVIEW

File: plugin/plugin_source.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/cloudquery/plugin-sdk/v4/glob"
 	"github.com/cloudquery/plugin-sdk/v4/message"
+	"github.com/cloudquery/plugin-sdk/v4/scheduler"
```
Comment (disq) [inline plugin/plugin_source.go:9]: Unsure if it's worth the extra import. This is the only line in the `plugin` package to refer to `scheduler`.

Comment (erezrokah) [reply plugin/plugin_source.go:9]: We can put this here https://github.com/cloudquery/plugin-sdk/tree/main/helpers

Comment (disq) [reply plugin/plugin_source.go:9]: `scheduler` already imports `helpers` (DFS uses `helpers.InterfaceSlice()` so it has to be a separate package. `opts` package, `func SchedulerOpts()`?

Comment (erezrokah) [reply plugin/plugin_source.go]: Now I see 
image

Comment (disq) [reply plugin/plugin_source.go]: forgot to `git add`, sorry fixed in https://github.com/cloudquery/plugin-sdk/pull/1900/commits/4d699f5bab90c9770e3f1e2766c39fca2b8b62b3

**Review by erezrokah — APPROVED** [review]

**Pull request merged**